### PR TITLE
Import wording

### PR DESCRIPTION
**Pull request by jameskerr**
This is what we've got now:

image

### CODE REVIEW

**Comment by philrz** [comment]
A proposed article to hyperlink from the word "docs" is up for review at #1626.

@jameskerr: Also, in your current screenshot I see you don't yet have hyperlinks attached to CSV, JSON, and Parquet, but I've got hyperlinks to them in the article, and I think it'd be good if you used them in the app as well. (Really, JSON and NDJSON are the ones I'm most concerned about because I suspect some users might not pick up on the subtle difference between the two, but I figure as long as we're being educational we might as well also be complete.)

**Comment by philrz** [comment]
@jameskerr: Now that #1626 is merged, [https://github.com/brimdata/brim/wiki/Importing-CSV-JSON-Parquet-and-ZST](https://github.com/brimdata/brim/wiki/Importing-CSV-JSON-Parquet-and-ZST) is the final place to link to.

**Comment by jameskerr** [comment]
Great I will add those.

**Comment by jameskerr** [comment]
I've updated the image in the description to show the changes.

File: app/lakes/import.tsx [hunk 1]
```diff
@@ -37,19 +37,32 @@ export default function TabImport() {
...
-              .zst
+            <Link href="https://github.com/brimdata/zed/blob/main/docs/formats/zson.md">
+              ZSON
             </Link>
           </p>
+          See{" "}
+          <Link href="https://github.com/brimdata/brim/wiki/Importing-CSV-JSON-Parquet-and-ZST">
+            docs
+          </Link>{" "}
+          for Zed platform support of{" "}
```
Comment (nwt) [inline app/lakes/import.tsx:58]: Nit: I think "support for" reads better than "support of".
```suggestion
          for Zed platform support for{" "}
```

**Review by nwt — COMMENTED** [review]
This looks good to me but I leave approval to @philrz.

**Review by philrz — APPROVED** [review]
Just tried it out in the branch and it looks great!

**Pull request merged**